tests/publiccloud/run_ltp.pm

```diff
@@ -85,7 +94,13 @@ sub cleanup {
     type_string('', terminate_with => 'ETX');
 
     upload_logs('ltp_log.raw', failok => 1);
-    parse_extra_log(LTP => 'ltp_log.json') if (script_run('test -f ltp_log.json') == 0);
+    parse_extra_log(LTP => "$root_dir/ltp_log.json") if (script_run("test -f $root_dir/ltp_log.json") == 0);
+
+    if (script_run("test -f $root_dir/stop_log_instance.sh") == 0) {
+        assert_script_run("bash $root_dir/stop_log_instance.sh");
+        assert_script_run("(cd /tmp/log_instance && tar -zcf $root_dir/instance_log.tar.gz *)");
```

Comment:
if I not missing something this would do the same , but looks more clean as for me :
```suggestion
        assert_script_run("tar -zcf $root_dir/instance_log.tar.gz /tmp/log_instance/*");
```

Reply:
In your case, the archive contains also `tmp/log_instance` folders. 
And I only wanna have one sub folder, which is the name/id of the instance.

Reply:
I would stick to my way of doing it.

Reply:
ok